_conferences/2025/02_call.html

```diff
@@ -77,9 +77,13 @@ <h3>Submissions</h3>
   
   <p>Accepted abstracts will be published as part of the conference program and a Book of Abstracts after the conference.</p> 
 
-  <p>Authors are invited to upload their anonymized submission for review to our Conftool website (link will follow).</p>
+  <p>Authors are invited to upload their anonymized submission for review to our Conftool website: <a href="https://www.conftool.net/mec2025/">https://www.conftool.net/mec2025/</a></p>
 
-  <p>The deadline for all submissions is <strong>13 December 2024</strong> (see <a href="#dates">IMPORTANT DATES</a> above). Conftool accepts abstracts as PDF files only. Since review will be anonymous, please ensure that all identifying information is removed from your PDF before submission.</p>
+  <p>The deadline for all submissions is <strong>13 December 2024</strong> (see <a href="#dates">IMPORTANT DATES</a> above). All submissions should include a title, author information (only in ConfTool!), a short abstract, up to five keywords and a PDF file of your proposal. Conftool accepts PDF files only. Since review will be anonymous, please ensure that all identifying   information is removed from your PDF before submission.</p>
```

Comment:
```suggestion
  <p>The deadline for all submissions is <strong>13 December 2024</strong> (see <a href="#dates">IMPORTANT DATES</a> above). All submissions should include a title, author information (only in ConfTool!), a short abstract, up to five keywords and a PDF file of your proposal. Conftool accepts PDF files only. Since review will be anonymous, please ensure that all identifying information is removed from your PDF before submission.</p>
```